### ISSUE TITLE
How are changes resolved if "merging" is "voting", is it a democracy?

### ISSUE THREAD

**Issue by ghost** [BelfordZ/starIPs#3]
The question more extended:

Using voting or majorities is not good for blockchains, but what happens when people want a change that was only merged by a minority of teams? What if 50% of teams accept and merge an ECIP and the other 50% doesn’t? If an ECIP has more than 50% of teams merging it, does that mean it is forced to the rest of the network participants?

**Comment by ghost** [comment]
My auto-answer to the question above:

It is not a democracy. It doesn't matter if a majority, the same number or minority of teams merge the same ECIP. The ones who decide in the end are the node operators and miners who have to achieve consensus at the network level. This means that they will chose the clients with the same rules they prefer, even if they were minority, a tie, or a majority.

In summary, the "voting" doesn't really exist, it is "rough consensus" or adoption of the rules that counts, not the amount of teams who merge them.

However, I expect that teams are going to tend to merge in tandem as they are competing for adoption and don't want to be left behind either by not adopting changes, or by adopting consensus rule changes nobody wants.
